### PR TITLE
Storyboard

### PR DESCRIPTION
**Pull request by daortizh**
- Executive Threat Briefing.
  Avoid to trigger click event on an opened comment
- Proxy's Incident Progression
  Fixed "Double root node" issue

### CODE REVIEW

**Comment by LedaLima** [comment]
LGTM

File: ui/proxy/js/components/IncidentProgressionPanel.react.js [hunk 1]
```diff
@@ -74,19 +74,21 @@ var IncidentProgressionPanel = React.createClass({
...
+        var requestRoot, referedRoot, node, canvasWidth, width, height, nodes, links;
 
-        // Build Requests root node
-        requestsRoot = d3.hierarchy({
-            id: this.state.root.id,
-            name: '',
-            type: this.state.root.type,
-            children: this.state.root.requests
-        });
+        /*leaves = Math.max(requestsRoot.leaves().length, referedRoot.leaves().length);
```
Comment (rabarona) [inline ui/proxy/js/components/IncidentProgressionPanel.react.js]: Is there a particular reason to keep this code commented? 
If so, there is one /\* starting and then in between a comment with // and finally a closing */. I would say treat everything as a big comment and remove the //. 

If it's not required I'd say remove the entire comment.

Comment (daortizh) [inline ui/proxy/js/components/IncidentProgressionPanel.react.js]: Not at this moment, that code is for a different approach. I'll remove it. Thanks

**Comment by rabarona** [comment]
LGTM

**Comment by EverLoSa** [comment]
LGTM

**Pull request merged**